helm/datalab/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.14.2
+version: 0.15
```

Comment:
Sorry, I think this should have a .0 on the end i.e. `0.15.0`! Sorry, that was my fault from my earlier comment! Once that is updated then I am happy for it to be merged though!